.github/actions/python/action.yaml

```diff
@@ -8,12 +8,24 @@ inputs:
 runs:
   using: "composite"
   steps:
+
     - name: Set up Python 3.9 for protos
+      uses: useblacksmith/setup-python@v6
+      # NOTE(hammadb) Blacksmith does not support windows, so we use the official action for windows and
+      # skip it on other platforms
+      if: runner.os != 'Windows'
+      with:
+        python-version: "3.9"
+        cache: "pip"
+        cache-dependency-path: "requirements*.txt"
+    - name: Set up Python 3.9 for protos (Windows)
+      if: runner.os == 'Windows'
       uses: actions/setup-python@v5
       with:
         python-version: "3.9"
```

Comment:
I guess we'd want to change it here too... must have existed before this PR, so up to you on whether it gets updated here or later.

Reply:
This is just how it was. I don't view this as a part of the scope of these changes.